src/components/SectionList/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React, {forwardRef} from 'react';
+import {SectionList as RNSectionList} from 'react-native';
+import ForwardedSectionList from './types';
+
+// eslint-disable-next-line react/function-component-definition
+const SectionList: ForwardedSectionList = (props, ref) => (
+    <RNSectionList
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...props}
+        ref={ref}
+    />
+);
+
+SectionList.displayName = 'SectionList';
```

Comment:
Shouldn't we keep same name as Android one?

Reply:
I didn't change it compared to the way it was before. Section list in index.js didn't have display name provided, so by default it had `SectionList`. On all platforms other than android `SectionList` component doesn't use forwardRef so it makes sense to have different displayNames

Reply:
But you passed the reference.